.github/workflows/security_scans.yml

```
@@ -0,0 +1,22 @@
+name: SAST Snyk Code
+on:
+  schedule:
+    - cron: '23 14 * * *'
+  push:
+    branches:
+      - 'main'
```

```suggestion
      - 'devel'
```
Only devel branch available

---

Done